lte/gateway/python/magma/pipelined/qos/qos_meter_impl.py

```diff
@@ -72,7 +72,7 @@ def add_qos(self, _, qos_info: QosInfo, parent=None) -> int:
         LOG.debug("Adding meter_id %d", meter_id)
         return meter_id
 
-    def remove_qos(self, meter_id: int, d, recovery_mode=False):
+    def remove_qos(self, meter_id: int, d, recovery_mode=False, _=False):
```

Comment:
_=False -> _

Reply:
it needs default value, since recovery_mode param has it.